coalib/bearlib/languages/definitions/markdown.py

```diff
@@ -0,0 +1,10 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class markdown:
```

Comment:
Same here. Change markdown into Markdown. Btw, Markdown is always written with an upper case M. That means you should also change your filename.